data-prepper-script/build.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'antlr'
+    id 'java'
+    id 'idea'
+}
+
+group 'org.opensearch.dataprepper'
+version '1.3.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
```

Comment:
I think this can all be removed since the parent project provides it. Same for the `test` section